xml/mysql_banners.xml

```diff
@@ -1065,6 +1311,38 @@
     <param pos="0" name="os.version" value="6.0"/>
     <param pos="0" name="os.cpe23" value="cpe:/o:debian:debian_linux:6.0"/>
   </fingerprint>
+  <fingerprint pattern="^(?:\d{1,2}\.\d{1,3}\.[a-f\d]{1,3}-)?(\d{1,2}\.\d{1,3}\.[a-f\d]{1,4})-MariaDB.+~zesty(?:-log)?$" flags="REG_ICASE">
+    <description>MariaDB MariaDB with Galera Cluster on Ubuntu 17.04 (zesty)</description>
```

Comment:
I don't think this one  or the following fingerprint is running Galera. Also, can you change the title and replace `zesty` with `Zesty Zapus`. Similar with `artful` -> `Aartful Ardvark` in the FP after it.  This will make the fingerprint descriptions more consistent.